packages/cli/src/utils/commands.ts

```diff
@@ -0,0 +1,8 @@
+import { spawnSync } from "node:child_process"
+
+// Retrieves the package manager based on the developer lockfile, using `ni`.
+export function getPreferredPackageManager() {
+  const agent = spawnSync("na", ['\?'], { encoding: 'utf8' }).stdout.trim()
```

Comment:
I got an error trying to trim() null. (see the main review message).

Reply:
@eduardoformiga this should be solved now under the latest version available.